bfgs.go

```diff
@@ -159,3 +159,7 @@ func (b *BFGS) NextDirection(loc *Location, dir []float64) (stepSize float64) {
 	floats.Scale(-1, dir)
 	return 1
 }
+
+func (_ BFGS) Requires() (grad, hess bool) {
```

Comment:
I was not sure if it was better to include _ or not. Omitting * was my mistake though. Will fix soon.